### PR TITLE
pds and poc testing executor updates

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
https://issues.jboss.org/browse/INTLY-1567

## Verification

Run the pipeline with the same config (different name): https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/ppaszki-poc-testing-executor/

### CODE REVIEW

File: jobs/integr8ly/pds-testing-executor.yaml [hunk 1]
```diff
@@ -65,85 +59,69 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
...
+                        if (TO_DO.contains('full')) {
                             install(false) // no sleep after install
                         } else {
                             println 'Install #3 skipped'
                         }
                     } // stage
+
+
+
+
```
Comment (tremes) [inline jobs/integr8ly/pds-testing-executor.yaml]: Unnecessary but that's really minor.

**Comment by tremes** [comment]
I think it looks good to me..I would have one further improvement (it can be a next issue or commit or whatever :)). Maybe if we define some variables with the pipeline names (as value) we will be able to refactor/optimize this pipeline (as well as the pds one ) further.

**Comment by pawelpaszki** [comment]
yeah. sounds good (further improvements). We can make it more flexible in terms of running tests as @trepel suggested. So I guess we can have it all in subsequent PR

**Comment by pawelpaszki** [comment]
@trepel @tremes any final comments?

**Comment by tremes** [comment]
Just merge it if it works :)

**Comment by pawelpaszki** [comment]
ok. I will merge it and will check, if tomorrow's PDS pipeline works the same.

**Pull request merged**